Bikeini/screens/AddBike.js

```diff
@@ -348,6 +348,10 @@ class AddBike extends React.Component {
       Alert.alert('Please set a location');
       return;
     }
+    if (!bikeData.title || !bikeData.brand || !bikeData.description) {
+      Alert.alert('Please make sure your ad contains a title, brand-information and description');
```

Comment:
The Alert-text is not displayed correctly (the text is cropped -> " /.../ title, brand-infor... ".
Could change from Alert to using red-text under the TextInputs like in LogIn and SignUp instead, but it could be a "waste" of time haha! 😸

Reply:
I personally feel like this doesn't have to be "fixed" so I gave my approval to the PR! 😄